python/activator/activator.py

```diff
@@ -225,28 +293,27 @@ def create_app():
         sys.exit(3)
 
 
-def dict_to_fanned_out_visit(obj, ctx):
-    """
-    Converts object literal(dict) to a Fanned Out instance.
-    Args:
-        ctx (SerializationContext): Metadata pertaining to the serialization
-            operation.
-        obj (dict): Object literal(dict)
-    """
-
-    if obj is None:
-        return None
-
-    return FannedOutVisit(**obj)
-
-
 def keda_start():
 
     try:
         setup_usdf_logger(
             labels={"instrument": instrument_name},
         )
 
+        # Prometheus gauge setup
+        instrument_name_gauge = instrument_name.lower()
```

Comment:
Backporting a fix from https://github.com/lsst-dm/next_visit_fan_out/commit/e067c61b59fc82be4352c2e797b52d9004047a17:
```suggestion
        instrument_name_gauge = instrument_name.lower().replace(" ", "_").replace("-", "_")
```